
```{js, echo = FALSE}
document.title = "Florian Privé"
```

<br>

### Publications

[**FULL LIST**](https://scholar.google.fr/citations?user=ykDpRgwAAAAJ)

- Privé, Florian, et al. "Identifying and correcting for misspecifications in GWAS summary statistics and polygenic scores." *Human Genetics and Genomics Advances* 3.4 (2022). [[Open Access]](https://doi.org/10.1016/j.xhgg.2022.100136)

- Privé, Florian. "Using the UK Biobank as a global reference of worldwide populations: application to measuring ancestry diversity from GWAS summary statistics." *Bioinformatics* 38.13 (2022): 3477-3480. [[Open Access]](https://doi.org/10.1093/bioinformatics/btac348)

- Privé, Florian, et al. "Portability of 245 polygenic scores when derived from the UK Biobank and applied to 9 ancestry groups from the same cohort." *The American Journal of Human Genetics* 109.1 (2022): 12-23. [[Open Access]](https://doi.org/10.1016/j.ajhg.2021.11.008)

- Privé, Florian. "Optimal linkage disequilibrium splitting." *Bioinformatics* 38.1 (2022): 255–256. [[Open Access]](https://doi.org/10.1093/bioinformatics/btab519)

- Privé, Florian, et al. "LDpred2: better, faster, stronger." *Bioinformatics* 36.22-23 (2020): 5424-5431. [[Open Access]](https://doi.org/10.1093/bioinformatics/btaa1029)

- Privé, Florian, et al. "Efficient toolkit implementing best practices for principal component analysis of population genetic data." *Bioinformatics* 36.16 (2020): 4449-4457. [[Open Access]](https://doi.org/10.1093/bioinformatics/btaa520)

- Privé, Florian, et al. "Performing highly efficient genome scans for local adaptation with R package pcadapt version 4." *Molecular Biology and Evolution* 37.7 (2020): 2153-2154. [[Open access]](https://doi.org/10.1093/molbev/msaa053)

- Privé, Florian, et al. "Making the most of Clumping and Thresholding for polygenic scores." *The American Journal of Human Genetics* 105.6 (2019): 1213-1221. [[Open access]](https://doi.org/10.1016/j.ajhg.2019.11.001)

- Privé, Florian, et al. "Efficient implementation of penalized regression for genetic risk prediction." *Genetics* 212.1 (2019): 65-74. [[Open access]](https://doi.org/10.1534/genetics.119.302019)

- Privé, Florian, et al. "Efficient analysis of large-scale genome-wide data with two R packages: bigstatsr and bigsnpr." *Bioinformatics* 34.16 (2018): 2781-2787. [[Open access]](https://doi.org/10.1093/bioinformatics/bty185)

<br>

### Conferences

- [invited] [SMPGD 2022](https://smpgd2022.sciencesconf.org/): Predicting complex traits and diseases from genetic data. [[Slides]](https://privefl.github.io/thesis-docs/smpgd22.html) 

- [APRS 2021](https://aacb.eventsair.com/aprs2021/): High-resolution portability of 245 polygenic scores when derived and applied in the same cohort. [[Recording]](https://youtu.be/NVOVZYeWqs4) [[Slides]](https://privefl.github.io/UKBB-PGS/APRS2021.html) 

- [e-Rum 2020](https://2020.erum.io/): Ultra fast penalized regressions with R package {bigstatsr}. [[Recording]](https://www.youtube.com/watch?v=sHlj_H8K2RI&t=127s) [[Slides]](https://privefl.github.io/R-presentation/slides-eRum2020.html) 

- [Rencontres R 2018](https://r2018-rennes.sciencesconf.org/): The R package bigstatsr: Memory- and Computation-Efficient Statistical Tools for Big Matrices [[Slides]](https://privefl.github.io/RR18/bigstatsr.html)

- [eRum 2018](https://2018.erum.io/): An R package for statistical tools with big matrices stored on disk. [[Recording]](https://youtu.be/w3a1-KLmDM8) [[Slides]](https://privefl.github.io/eRum-2018/slides.html)

- [Recomb-Genetics 2018](http://recomb2018.fr/recomb-genetics/): Predicting complex diseases: performance and robustness. [[Slides]](https://privefl.github.io/thesis-docs/recomb18.html)

- [LIFE 2018](https://life.univ-grenoble-alpes.fr/life-world/events-news/life-annual-meeting-725690.htm?RH=11210192165071361): Predicting complex diseases: performance and robustness. [[Slides]](https://github.com/privefl/paper2-PRS/blob/master/LIFE/pres-f-priv%C3%A9.pdf)

- [hackseq 2017](http://www.hackseq.com/): Developing advanced R tutorials for genomic data analysis. [[Website]](https://hackseq.github.io/2017_project_5/)

- [useR!2017](https://user2017.brussels/): The R package bigstatsr:
Memory- and Computation-Efficient Tools for Big Matrices. [[Recording]](https://t.co/aYt0q8MeXJ) [[Slides]](https://privefl.github.io/useR-2017/slides.html)

- [ALT'2016](http://alt2016.sciencesconf.org/): Goodness-of-fit tests for the Weibull distribution with censored data. [[Slides]](slides_GOFW.pdf#1)

<br>

### Some other presentations

You can reuse (and modify) some of my presentations as long as you give attribution (just as in open source licensing).

- [Memory- and Computation-Efficient Statistical Tools for Big Matrices](https://privefl.github.io/R-presentation/bigrverse.html) ([R-Lille RUG](https://t.co/fzuM15GBCT)) [[Recording]](https://www.youtube.com/watch?v=UXKYqpAo6cs)

- [Predicting complex traits and diseases from genetic data](https://privefl.github.io/thesis-docs/lecture-KI.html) ([Lecture at Karolinska Institute](https://kiwas.ki.se/katalog/katalog/kurs/9304))

- [Efficient penalized regression methods for genetic prediction](https://privefl.github.io/thesis-docs/PLR-genetics.html) ([Prediction Modelling Presentations at KCL](https://www.maudsleybrc.nihr.ac.uk/facilities/prediction-modelling-group/))

- [Agile Research](https://privefl.github.io/thesis-docs/agile-research.html) ([NCRR](https://econ.au.dk/the-national-centre-for-register-based-research/))

- [Penalized methods for genetic data analysis](https://privefl.github.io/R-presentation/penalised-genetics.html) ([Lecture at King's College London](https://www.kcl.ac.uk/ioppn/depts/biostatisticshealthinformatics/teaching/courses/machine-learning))

- [R Markdown](https://privefl.github.io/R-presentation/rmarkdown.html) ([Grenoble RUG](https://r-in-grenoble.github.io/))

- [R beyond statistics](https://privefl.github.io/R-presentation/beyond-stats.html) ([SARI, Grenoble](https://sari.grenoble-inp.fr/doku.php))

- Efficient analysis of large-scale matrices with two R packages: bigstatsr and bigsnpr: [presentation](https://privefl.github.io/R-presentation/bigstatsr.html) + [exercise](https://privefl.github.io/bigsnpr/articles/exo.html) ([State of The R RUG](https://stateofther.github.io/))

- [Efficient statistical tools for analyzing omics data, with a focus on polygenic risk prediction](https://privefl.github.io/thesis-docs/aarhus.html) (Aarhus, Denmark)

- [What R can do for you](https://privefl.github.io/R-presentation/whatrcando.html) ([Grenoble RUG](https://r-in-grenoble.github.io/))

- [High-dimensional data: a different kind of big data](https://privefl.github.io/thesis-docs/data-club.html) ([Grenoble Data Club](https://data-institute.univ-grenoble-alpes.fr/education/data-club-730304.htm))

- [Make your personal webpage with R](https://privefl.github.io/R-presentation/rmarkdown-websites.html) ([Grenoble RUG](https://r-in-grenoble.github.io/))

- [Instructor Software Carpentry](https://grenoble-alpes-data-institute.github.io/2018-05-31-grenoble-software-carpentry/)

- [The R package {bigstatsr}: memory- and computation-efficient tools for big matrices stored on disk](https://privefl.github.io/R-presentation/bigstatsr.html) ([Grenoble RUG](https://r-in-grenoble.github.io/))

- [Rcpp](https://privefl.github.io/R-presentation/Rcpp.html) ([Grenoble RUG](https://r-in-grenoble.github.io/))

- [RStudio & Git](https://privefl.github.io/R-presentation/rstudio-git.html) ([Grenoble RUG](https://r-in-grenoble.github.io/))

- [Visualisation with ggplot2](https://privefl.github.io/R-presentation/ggplot2.html) ([Grenoble RUG](https://r-in-grenoble.github.io/))

<br>

### Teaching

- advanced R course for PhD students (141H in five years); see [online materials](https://privefl.github.io/advr38book/)

- introductory R course for PhD students (21H)

- statistical methods for first year engineer students (third year after high school) at ENSIMAG (18H)

- lectures and practical exercises in Mathematics for students in their first semester after high school (128H in two years)

- software carpentry instructor -- introductory R course (7H)
