<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>com.knit</title>

<script src="site_libs/header-attrs-2.11/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/readable.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>








<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-inverse  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Florian Privé</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="about.html">
    <span class="fa fa-user"></span>
     
    About
  </a>
</li>
<li>
  <a href="com.html">Papers / Talks</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    #rstats
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="https://privefl.github.io/blog/">
        <span class="fa fa-rss"></span>
         
        Blog
      </a>
    </li>
    <li>
      <a href="https://privefl.github.io/bigstatsr/">
        <span class="fa fa-archive"></span>
         
        Package bigstatsr
      </a>
    </li>
    <li>
      <a href="https://privefl.github.io/bigsnpr/">
        <span class="fa fa-archive"></span>
         
        Package bigsnpr
      </a>
    </li>
    <li>
      <a href="https://r-in-grenoble.github.io/">
        <span class="fa fa-users"></span>
         
        #rusersgrenoble
      </a>
    </li>
    <li>
      <a href="https://privefl.github.io/advr38book/">
        <span class="fa fa-graduation-cap"></span>
         
        Advanced R course
      </a>
    </li>
  </ul>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="mailto:florian.prive.21@gmail.com">
    <span class="fa fa-envelope-o fa"></span>
     
  </a>
</li>
<li>
  <a href="https://github.com/privefl/privefl.github.io">
    <span class="fa fa-github fa"></span>
     
  </a>
</li>
<li>
  <a href="https://twitter.com/privefl">
    <span class="fa fa-twitter fa"></span>
     
  </a>
</li>
<li>
  <a href="http://stackoverflow.com/users/6103040/f-privé">
    <span class="fa fa-stack-overflow fa"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">




</div>


<script type="text/javascript">
document.title = "Florian Privé"
</script>
<p><br></p>
<div id="publications" class="section level3">
<h3>Publications</h3>
<p><a href="https://scholar.google.fr/citations?user=ykDpRgwAAAAJ"><strong>FULL LIST</strong></a></p>
<ul>
<li><p>Privé, Florian, et al. “Identifying and correcting for misspecifications in GWAS summary statistics and polygenic scores.” <em>bioRxiv</em> (2022). <a href="https://doi.org/10.1101/2021.03.29.437510">[preprint]</a></p></li>
<li><p>Privé, Florian. “Using the UK Biobank as a global reference of worldwide populations: application to measuring ancestry diversity from GWAS summary statistics.” <em>bioRxiv</em> (2021). <a href="https://doi.org/10.1101/2021.10.27.466078">[preprint]</a></p></li>
<li><p>Privé, Florian, et al. “Portability of 245 polygenic scores when derived from the UK Biobank and applied to 9 ancestry groups from the same cohort.” <em>The American Journal of Human Genetics</em> 109.1 (2022): 12-23. <a href="https://doi.org/10.1016/j.ajhg.2021.11.008">[Paper]</a></p></li>
<li><p>Privé, Florian. “Optimal linkage disequilibrium splitting.” <em>Bioinformatics</em> 38.1 (2022): 255–256. <a href="https://doi.org/10.1093/bioinformatics/btab519">[Open Access]</a></p></li>
<li><p>Privé, Florian, et al. “LDpred2: better, faster, stronger.” <em>Bioinformatics</em> 36.22-23 (2020): 5424-5431. <a href="https://doi.org/10.1093/bioinformatics/btaa1029">[Open Access]</a></p></li>
<li><p>Privé, Florian, et al. “Efficient toolkit implementing best practices for principal component analysis of population genetic data.” <em>Bioinformatics</em> 36.16 (2020): 4449-4457. <a href="https://doi.org/10.1093/bioinformatics/btaa520">[Open Access]</a></p></li>
<li><p>Privé, Florian, et al. “Making the most of Clumping and Thresholding for polygenic scores.” <em>The American Journal of Human Genetics</em> 105.6 (2019): 1213-1221. <a href="https://doi.org/10.1016/j.ajhg.2019.11.001">[Open access]</a></p></li>
<li><p>Privé, Florian, et al. “Efficient implementation of penalized regression for genetic risk prediction.” <em>Genetics</em> 212.1 (2019): 65-74. <a href="https://doi.org/10.1534/genetics.119.302019">[Open access]</a></p></li>
<li><p>Privé, Florian, et al. “Efficient analysis of large-scale genome-wide data with two R packages: bigstatsr and bigsnpr.” <em>Bioinformatics</em> 34.16 (2018): 2781-2787. <a href="https://doi.org/10.1093/bioinformatics/bty185">[Open access]</a></p></li>
</ul>
<p><br></p>
</div>
<div id="conferences" class="section level3">
<h3>Conferences</h3>
<ul>
<li><p>[invited] <a href="https://smpgd2022.sciencesconf.org/">SMPGD 2022</a>: Predicting complex traits and diseases from genetic data. <a href="https://privefl.github.io/thesis-docs/smpgd22.html">[Slides]</a></p></li>
<li><p><a href="https://aacb.eventsair.com/aprs2021/">APRS 2021</a>: High-resolution portability of 245 polygenic scores when derived and applied in the same cohort. <a href="https://youtu.be/NVOVZYeWqs4">[Recording]</a> <a href="https://privefl.github.io/UKBB-PGS/APRS2021.html">[Slides]</a></p></li>
<li><p><a href="https://2020.erum.io/">e-Rum 2020</a>: Ultra fast penalized regressions with R package {bigstatsr}. <a href="https://www.youtube.com/watch?v=sHlj_H8K2RI&amp;t=127s">[Recording]</a> <a href="https://privefl.github.io/R-presentation/slides-eRum2020.html">[Slides]</a></p></li>
<li><p><a href="https://r2018-rennes.sciencesconf.org/">Rencontres R 2018</a>: The R package bigstatsr: Memory- and Computation-Efficient Statistical Tools for Big Matrices <a href="https://privefl.github.io/RR18/bigstatsr.html">[Slides]</a></p></li>
<li><p><a href="https://2018.erum.io/">eRum 2018</a>: An R package for statistical tools with big matrices stored on disk. <a href="https://youtu.be/w3a1-KLmDM8">[Recording]</a> <a href="https://privefl.github.io/eRum-2018/slides.html">[Slides]</a></p></li>
<li><p><a href="http://recomb2018.fr/recomb-genetics/">Recomb-Genetics 2018</a>: Predicting complex diseases: performance and robustness. <a href="https://privefl.github.io/thesis-docs/recomb18.html">[Slides]</a></p></li>
<li><p><a href="https://life.univ-grenoble-alpes.fr/life-world/events-news/life-annual-meeting-725690.htm?RH=11210192165071361">LIFE 2018</a>: Predicting complex diseases: performance and robustness. <a href="https://github.com/privefl/paper2-PRS/blob/master/LIFE/pres-f-priv%C3%A9.pdf">[Slides]</a></p></li>
<li><p><a href="http://www.hackseq.com/">hackseq 2017</a>: Developing advanced R tutorials for genomic data analysis. <a href="https://hackseq.github.io/2017_project_5/">[Website]</a></p></li>
<li><p><a href="https://user2017.brussels/">useR!2017</a>: The R package bigstatsr: Memory- and Computation-Efficient Tools for Big Matrices. <a href="https://t.co/aYt0q8MeXJ">[Recording]</a> <a href="https://privefl.github.io/useR-2017/slides.html">[Slides]</a></p></li>
<li><p><a href="http://alt2016.sciencesconf.org/">ALT’2016</a>: Goodness-of-fit tests for the Weibull distribution with censored data. <a href="slides_GOFW.pdf#1">[Slides]</a></p></li>
</ul>
<p><br></p>
</div>
<div id="some-other-presentations" class="section level3">
<h3>Some other presentations</h3>
<p>You can reuse (and modify) some of my presentations as long as you give attribution (just as in open source licensing).</p>
<ul>
<li><p><a href="https://privefl.github.io/R-presentation/bigrverse.html">Memory- and Computation-Efficient Statistical Tools for Big Matrices</a> (<a href="https://t.co/fzuM15GBCT">R-Lille RUG</a>) <a href="https://www.youtube.com/watch?v=UXKYqpAo6cs">[Recording]</a></p></li>
<li><p><a href="https://privefl.github.io/thesis-docs/lecture-KI.html">Predicting complex traits and diseases from genetic data</a> (<a href="https://kiwas.ki.se/katalog/katalog/kurs/9304">Lecture at Karolinska Institute</a>)</p></li>
<li><p><a href="https://privefl.github.io/thesis-docs/PLR-genetics.html">Efficient penalized regression methods for genetic prediction</a> (<a href="https://www.maudsleybrc.nihr.ac.uk/facilities/prediction-modelling-group/">Prediction Modelling Presentations at KCL</a>)</p></li>
<li><p><a href="https://privefl.github.io/thesis-docs/agile-research.html">Agile Research</a> (<a href="https://econ.au.dk/the-national-centre-for-register-based-research/">NCRR</a>)</p></li>
<li><p><a href="https://privefl.github.io/R-presentation/penalised-genetics.html">Penalized methods for genetic data analysis</a> (<a href="https://www.kcl.ac.uk/ioppn/depts/biostatisticshealthinformatics/teaching/courses/machine-learning">Lecture at King’s College London</a>)</p></li>
<li><p><a href="https://privefl.github.io/R-presentation/rmarkdown.html">R Markdown</a> (<a href="https://r-in-grenoble.github.io/">Grenoble RUG</a>)</p></li>
<li><p><a href="https://privefl.github.io/R-presentation/beyond-stats.html">R beyond statistics</a> (<a href="https://sari.grenoble-inp.fr/doku.php">SARI, Grenoble</a>)</p></li>
<li><p>Efficient analysis of large-scale matrices with two R packages: bigstatsr and bigsnpr: <a href="https://privefl.github.io/R-presentation/bigstatsr.html">presentation</a> + <a href="https://privefl.github.io/bigsnpr/articles/exo.html">exercise</a> (<a href="https://stateofther.github.io/">State of The R RUG</a>)</p></li>
<li><p><a href="https://privefl.github.io/thesis-docs/aarhus.html">Efficient statistical tools for analyzing omics data, with a focus on polygenic risk prediction</a> (Aarhus, Denmark)</p></li>
<li><p><a href="https://privefl.github.io/R-presentation/whatrcando.html">What R can do for you</a> (<a href="https://r-in-grenoble.github.io/">Grenoble RUG</a>)</p></li>
<li><p><a href="https://privefl.github.io/thesis-docs/data-club.html">High-dimensional data: a different kind of big data</a> (<a href="https://data-institute.univ-grenoble-alpes.fr/education/data-club-730304.htm">Grenoble Data Club</a>)</p></li>
<li><p><a href="https://privefl.github.io/R-presentation/rmarkdown-websites.html">Make your personal webpage with R</a> (<a href="https://r-in-grenoble.github.io/">Grenoble RUG</a>)</p></li>
<li><p><a href="https://grenoble-alpes-data-institute.github.io/2018-05-31-grenoble-software-carpentry/">Instructor Software Carpentry</a></p></li>
<li><p><a href="https://privefl.github.io/R-presentation/bigstatsr.html">The R package {bigstatsr}: memory- and computation-efficient tools for big matrices stored on disk</a> (<a href="https://r-in-grenoble.github.io/">Grenoble RUG</a>)</p></li>
<li><p><a href="https://privefl.github.io/R-presentation/Rcpp.html">Rcpp</a> (<a href="https://r-in-grenoble.github.io/">Grenoble RUG</a>)</p></li>
<li><p><a href="https://privefl.github.io/R-presentation/rstudio-git.html">RStudio &amp; Git</a> (<a href="https://r-in-grenoble.github.io/">Grenoble RUG</a>)</p></li>
<li><p><a href="https://privefl.github.io/R-presentation/ggplot2.html">Visualisation with ggplot2</a> (<a href="https://r-in-grenoble.github.io/">Grenoble RUG</a>)</p></li>
</ul>
<p><br></p>
</div>
<div id="teaching" class="section level3">
<h3>Teaching</h3>
<ul>
<li><p>advanced R course for PhD students (111H in four years); see <a href="https://privefl.github.io/advr38book/">online materials</a></p></li>
<li><p>introductory R course for PhD students (21H)</p></li>
<li><p>statistical methods for first year engineer students (third year after high school) at ENSIMAG (18H)</p></li>
<li><p>lectures and practical exercises in Mathematics for students in their first semester after high school (128H in two years)</p></li>
<li><p>software carpentry instructor – introductory R course (7H)</p></li>
</ul>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
